specification/wse/src/main/scripts/org/kaazing/specification/wse/data/binary.as.escaped.text/echo.escaped.characters/request.rpt

```diff
@@ -0,0 +1,55 @@
+property sequence ${wse:randomInt(100)}
+
+connect http://localhost:8080/path/;e/cte?query
+connected
+
+write method "POST"
+write version "HTTP/1.1"
+write header host
+write header content-length
+write header "X-WebSocket-Version" "wseb-1.0"
+write header "X-Sequence-No" ${wse:asString(sequence)}
+write close
+
+read status "201" /.+/
+read version "HTTP/1.1"
+read header "Content-Type" /text\/plain;charset=(?i)utf-8/
+
+read /(?<upstream>http:\/\/localhost:8080\/path\/.+)/ "\n"
+read /(?<downstream>http:\/\/localhost:8080\/path\/.+)/ "\n"
+read notify CREATED
+read closed
+
+# Downstream
+connect await CREATED
+connect ${downstream}
+connected
+
+write method "GET"
+write version "HTTP/1.1"
+write header host
+write header "X-Sequence-No" ${wse:asString(sequence + 1)}
+write header content-length
+write close
+
+read status "200" /.+/
+read header "Content-Type" "text/plain;charset=windows-1252"
+read header "Connection" "close"
+
+read [0x80 0x04 0x7f 0x30 0x7f 0x72 0x7f 0x6e 0x7f 0x7f]
```

Comment:
Perhaps I'll add that to the spec.  The 0xC2 0x80 that is below is in the charset=utf-8 section where 0x80 needs extended bytes to represent it